backend/services/badge_awarding_service.py

```
@@ -9,6 +9,14 @@ class BadgeAwardingService:
     def __init__(self, user_id):
         self.user_id = user_id  # The user to whom the badge is awarded
 
+    def check_and_award_badges(self, event_type, **kwargs):
```

Ruff (0.8.2)

Missing return type annotation for public function `check_and_award_badges`

Add return type annotation: `None`

(ANN201)